scripts/aznfs_install.sh

```diff
@@ -183,73 +336,48 @@ esac
 
 ensure_pkg "wget"
 
+if [ "$RUN_MODE" == "auto-update" ]; then
+    parse_config_file
+
+    # Define the GitHub API URL to get the latest release
+    API_URL="https://api.github.com/repos/$REPO_OWNER/$REPO_NAME/releases/latest"
+    # RELEASE_NUMBER=$(curl -s "$API_URL" | grep "tag_name" | cut -d '"' -f 4)
+    RELEASE_NUMBER="0.1.199"
```

Comment:
this needs to be uncommented.
Once you uncomment it, then you should also check that the above pipeline yielded a valid RELEASE_NUMBER.
It might be better to make the curl separate so that we can check the return status of that as that's a n/w operation and can likely fail. If it fails, we should not proceed